### PR TITLE
feat(coffee): status command

### PR DESCRIPTION
**Pull request by erics118**
## Description

made a status command to see the current caffeinate status
the menu bar command no longer has its subtitle update to show the status

## Screencast

CleanShot 2023-11-11 at 23 48 58

CleanShot 2023-11-11 at 23 49 06

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/coffee/src/index.tsx [hunk 1]
```diff
@@ -1,44 +1,35 @@
...
-import { useEffect } from "react";
 import { stopCaffeinate, startCaffeinate } from "./utils";
 
-export default function Command() {
-  const hasLaunchContext = environment.launchContext?.caffinated !== undefined;
+export default function Command(props: LaunchProps) {
+  const hasLaunchContext = props.launchContext?.caffeinated !== undefined;
 
-  const { isLoading, data, mutate } = useExec("ps aux | pgrep caffeinate", [], {
+  const { isLoading, data, mutate } = useExec("pgrep caffeinate", [], {
```
Comment (erics118) [inline extensions/coffee/src/index.tsx:8]: I think the original intention was for it to be `ps aux | grep caffeinate`, but `pgrep caffeinate` does the same thing.

File: extensions/coffee/src/utils.ts [hunk 2]
```diff
@@ -32,7 +31,8 @@ function preventArguments(args?: string | undefined) {
 export async function stopCaffeinate(updateMenubar = true, hudMessage?: string) {
   if (updateMenubar) {
     try {
-      await launchCommand({ name: "index", type: LaunchType.Background, context: { caffinated: false } });
+      await launchCommand({ name: "index", type: LaunchType.Background, context: { caffeinated: false } });
+      await launchCommand({ name: "status", type: LaunchType.Background, context: { caffeinated: false } });
```
Comment (erics118) [inline extensions/coffee/src/utils.ts:35]: Now, also immediately updates the status of the new `status` command. The `index` command is now solely responsible for the menu bar item

**Comment by pernielsentikaer** [comment]
Hi @erics118 🔥 

You're on fire! Thanks for another awesome update 🚀 

I can't enable it from Menubar (anymore?) but only disable it, do you have the same experience?

**Comment by erics118** [comment]
Oops, weird bug with preventing the `index` command from calling itself. Fixed now

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**